src/bentoml/_internal/io_descriptors/json.py

```diff
@@ -280,7 +280,7 @@ def openapi_example(self):
                 self.sample
             ):
                 return self.sample.dict()
-            elif isinstance(self.sample, str):
+            elif isinstance(self.sample, (str, list)):
```

Comment:
Shouldn't this addition of `list` happen further down on l. 292? We want to support `list` in addition to `dict`, not in addition to `str`.

Reply:
You have to dump the list with JSON. This line ensures list of a complex datatype can be serialized.

Reply:
@aarnphm Could you provide some examples here?

Reply:
similar to the one that @mqk provides:
```python
@svc.api(input=JSON.from_sample([{"a": 1, "b": 2}, {"a": 2, "b": 4}]), output=JSON())
```